mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/scaladsl/MqttSession.scala

```diff
@@ -8,6 +8,7 @@ package scaladsl
 import java.util.concurrent.atomic.AtomicLong
 
 import akka.actor.ExtendedActorSystem
+import akka.actor.typed.Props
 import akka.actor.typed.internal.adapter.{ActorRefAdapter, PropsAdapter}
```

Comment:
Instead, Alpakka should use the public API version of `PropsAdapter` and the `ActorRefAdapter` shouldn't be used, either.

Reply:
Done.